CMakeLists.txt

```diff
@@ -40,49 +40,49 @@ option(WITH_TEEM "Adds support of nrrd files through teem" OFF)
 
 
 set(source_files
```

Comment:
I may be wrong, but I think it is more common to set up CMake in a "recursive fashion", so your root `CMakeLists.txt`  has something like
```
add_subdirectory(src)
```
and each subdirectory has it's own `CMakeList` that includes another directory or the sources....

 See these links:
https://cliutils.gitlab.io/modern-cmake/chapters/basics/structure.html
https://gitlab.com/CLIUtils/modern-cmake/-/blob/master/examples/extended-project/CMakeLists.txt

How do you feel about that organization?

Reply:
Same as in the other issue. Create an additional issue to fix cmake code.

Reply:
Sounds good